components/binding-usage-controller/docs/assets/architecture.html

```diff
@@ -2,10 +2,11 @@
 <!DOCTYPE html>
```

Comment:
there is a new rule about the diagrams:

>  Export the diagram as an SVG and save it under the corresponding assets directory.

source: https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/diagrams.md#tool

Because you already changing that part then IMO is worth to align with new rule and repalce  *html and *png files with *svg